app/Providers/TelescopeServiceProvider.php

```
@@ -0,0 +1,64 @@
+    /**
+     * Register the Telescope gate.
+     *
+     * This gate determines who can access Telescope in non-local environments.
+     */
+    protected function gate(): void
+    {
+        Gate::define('viewTelescope', function ($user) {
+            return in_array($user->email, [
+                //
+            ]);
+        });
```

PHPMD (2.15.0)

Avoid using static access to class '\Illuminate\Support\Facades\Gate' in method 'gate'. (Clean Code Rules)

(StaticAccess)